### PR TITLE
correct spelling of 'take effect' on menus and comments. Signed-off-by: John Bentley <johnbentley.public@gmail.com>

### PR DESCRIPTION
**Pull request by mathslug**
I saw this when setting up my install so I thought I'd open a PR. I hope this is helpful. Thank you for taking a look. I will also submit a bug report with a patch on https://bugs.freebsd.org and link out to this PR.

### CODE REVIEW

File: contrib/sendmail/contrib/buildvirtuser [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/perl -w
 
 # Copyright (c) 1999-2004, 2007 Gregory Neil Shapiro.  All Rights Reserved.
-# 
+#
```
Comment (mathslug) [inline contrib/sendmail/contrib/buildvirtuser:4]:
```suggestion
# 
```

File: contrib/sendmail/contrib/buildvirtuser [hunk 2]
```diff
@@ -78,10 +78,10 @@ Here is an example file from the /etc/mail/virtusers/ directory:
  MAILER-DAEMON	gshapiro+bounce.$DOMAIN@example.net
  postmaster	gshapiro+$LHS.$DOMAIN@example.net
  webmaster	gshapiro+$LHS.$DOMAIN@example.net
- 
+
```
Comment (mathslug) [inline contrib/sendmail/contrib/buildvirtuser:81]:
```suggestion
 
```

File: contrib/sendmail/contrib/buildvirtuser [hunk 3]
```diff
@@ -78,10 +78,10 @@ Here is an example file from the /etc/mail/virtusers/ directory:
...
  # Defaults
  		error:nouser No such user
- 
+
```
Comment (mathslug) [inline contrib/sendmail/contrib/buildvirtuser:84]:
```suggestion
 
```

**Comment by mathslug** [comment]
Just submitted bug number 259636 with a diff and more detail.

File: sys/dev/mlx4/mlx4_core/mlx4_cmd.c [hunk 4]
```diff
@@ -3286,4 +3286,3 @@ int mlx4_vf_set_enable_smi_admin(struct mlx4_dev *dev, int slave, int port,
 	return 0;
 }
 EXPORT_SYMBOL_GPL(mlx4_vf_set_enable_smi_admin);
```
Comment (mathslug) [inline sys/dev/mlx4/mlx4_core/mlx4_cmd.c:3288]:
```suggestion
EXPORT_SYMBOL_GPL(mlx4_vf_set_enable_smi_admin);

```

**Comment by emaste** [comment]
Thanks for the submission. Some notes:

- anything under contrib/ should be sent to the upstream project(s) first, to be brought in with the next update
  - exception perhaps if upstream is unresponsive and it's a user-facing issue
  - I see you have a "remove contrib edits" change; would you please rebase these changes so that there's just one commit that can be brought in
  - please do not include unrelated patches (whitespace shuffling, etc.)
- we have not yet rolled this out formally, but would suggest reading https://developercertificate.org/ and adding a `Signed-off-by:` to the commit indicating that you agree

**Comment by mathslug** [comment]
Thanks for the feedback @emaste. I believe I've addressed everything and this is ready for another look.

**Comment by bsdimp** [comment]
landed as f7c32ed617858bcd22f8d1b03199099d50125721

**Pull request closed without merging**